--- src/Makefile.am	2008-10-26 15:36:56.000000000 +0100
+++ src/Makefile.am.oden	2008-11-29 14:38:01.000000000 +0100
@@ -3,7 +3,7 @@
 
 lib_LTLIBRARIES = libmemcached_functions_mysql.la
 libmemcached_functions_mysql_la_SOURCES = args.c add.c get.c set.c prepend.c replace.c increment.c decrement.c append.c delete.c servers.c version.c stats.c
-libmemcached_functions_mysql_la_LDFLAGS = -module
+libmemcached_functions_mysql_la_LDFLAGS = -module -avoid-version
 # had to add this back
 libmemcached_functions_mysql_la_LIBADD = $(DEPS_LIBS)
 
